SConstruct

```diff
@@ -5,8 +5,17 @@ import platform
 import subprocess
 import time
 import errno
+# pylint: disable=no-name-in-module
+# pylint: disable=import-error
+import SCons.Warnings
+# pylint: enable=no-name-in-module
+# pylint: enable=import-error
 from SCons.Script import BUILD_TARGETS
 
+# pylint: disable=no-member
+SCons.Warnings.warningAsException()
```

Comment:
a better way to deal with these scons warnings is to define corresponding things in utils/sl/fake_scons/ so that pylint (which knows nothing about scons) assumes it's good